ci/appveyor.yml

```diff
@@ -0,0 +1,261 @@
+
+    echo "DT_BUILD_SUFFIX = $env:DT_BUILD_SUFFIX"
+
+
+
+    if ($env:DT_RELEASE -eq "True") {
+
+      # =======================================================================
+      #   Build and test wheel for Python 3.6
+      # =======================================================================
+
+      $env:PATH = "C:/Python36-x64;C:/Python36-x64/Scripts;$DEFAULT_PATH"
+
+      python ci/ext.py wheel
+
+      $DT_WHEEL = ls dist/*-cp36-*.whl
+
+      echo "----- _build_info.py for Python 3.6 ------------------------------"
+
+      cat src/datatable/_build_info.py
+
+      echo "------------------------------------------------------------------"
+
+      python -m pip install --upgrade pip
+
+      pip install $DT_WHEEL pytest docutils pandas
+
+      pytest -ra --maxfail=10 -Werror -vv --showlocals ./tests/
+
+
+
+      # =======================================================================
+      #   Build and test wheel for Python 3.7
+      # =======================================================================
+
+      $env:PATH = "C:/Python37-x64;C:/Python37-x64/Scripts;$DEFAULT_PATH"
+
+      python ci/ext.py wheel
+
+      $DT_WHEEL = ls dist/*-cp37-*.whl
+
+      echo "----- _build_info.py for Python 3.7 ------------------------------"
+
+      cat src/datatable/_build_info.py
+
+      echo "------------------------------------------------------------------"
+
+      python -m pip install --upgrade pip
+
+      pip install $DT_WHEEL pytest docutils pandas
+
+      pytest -ra --maxfail=10 -Werror -vv --showlocals ./tests/
+
+    }
+
+
+
+    # =======================================================================
+    #   Build and test wheel for Python 3.8
+    # =======================================================================
+
+    $env:PATH = "C:/Python38-x64;C:/Python38-x64/Scripts;$DEFAULT_PATH"
+
+    python ci/ext.py wheel
+
+    $DT_WHEEL = ls dist/*-cp38-*.whl
+
+    echo "----- _build_info.py for Python 3.8 ------------------------------"
+
+    cat src/datatable/_build_info.py
+
+    echo "------------------------------------------------------------------"
+
+    python -m pip install --upgrade pip
+
+    pip install $DT_WHEEL pytest docutils pandas
+
+    pytest -ra --maxfail=10 -Werror -vv --showlocals ./tests/
+
+
+
+    # =======================================================================
+    #   Build and test wheel for Python 3.9
+    # =======================================================================
+
+    $env:PATH = "C:/Python39-x64;C:/Python39-x64/Scripts;$DEFAULT_PATH"
+
+    python ci/ext.py wheel
+
+    $DT_WHEEL = ls dist/*-cp39-*.whl
+
+    echo "----- _build_info.py for Python 3.9 ------------------------------"
+
+    cat src/datatable/_build_info.py
+
+    echo "------------------------------------------------------------------"
+
+    python -m pip install --upgrade pip
+
+    pip install $DT_WHEEL pytest docutils pandas

```

Comment:
In theory, `pip` and `python -m pip` could invoke different versions of pip package.
While it's unlikely to be happening here, I'm not sure what's the reason of using 2 different call conventions.

Reply:
Just happened with no particular reason.